sdk/aqueduct/client.py

```diff
@@ -552,6 +556,9 @@ def publish_flow(
             source_flow:
                 Used to identify the source flow for this flow. This can be identified
                 via an object (Flow), name (str), or id (str or uuid).
+            use_local:
+                Used to identify if local data is used. In order to use `LocalData`, this has to be set
```

Comment:
Maybe rephrase: "Must be set if any artifact in the flow is derived from local data."